Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM debian:stretch
+
+USER root
+RUN apt-get update -y && \
+	apt-get -y install build-essential libirrlicht-dev cmake libbz2-dev libpng-dev libjpeg-dev \
+		libsqlite3-dev libcurl4-gnutls-dev zlib1g-dev libgmp-dev libjsoncpp-dev git && \
+		apt-get clean && rm -rf /var/cache/apt/archives/* && \
+		rm -rf /var/lib/apt/lists/*
+
+COPY . /usr/src/minetest
+
+RUN	mkdir -p /usr/src/minetest/cmakebuild && cd /usr/src/minetest/cmakebuild && \
+    	cmake -DCMAKE_INSTALL_PREFIX=/usr/local -DCMAKE_BUILD_TYPE=Release -DRUN_IN_PLACE=FALSE \
+		-DENABLE_GETTEXT=TRUE \
+		-DBUILD_SERVER=TRUE \
+		-DBUILD_CLIENT=FALSE \
+		-DENABLE_SYSTEM_JSONCPP=1 \
+		-DENABLE_SOUND=0 \
```

Comment:
probably not needed since client is disabled anyway

Reply:
BUILD_CLIENT is set to true by default. BUILD_SERVER is set to FALSE by default